### ISSUE TITLE
Add timestamp and user when fixing empty {{AfC submission}} template

### ISSUE THREAD

**Issue by robertsky** [WPAFC/afch-rewrite#173]
example: https://en.wikipedia.org/w/index.php?title=Draft:Frank_Stubbs_Promotes_(TV_Series)&type=revision&diff=1037549705&oldid=1037331040&diffmode=source

**Comment by primefac** [comment]
Genuinely curious, which timestamp would it use? The time of the reviewer or the time it was submitted? Anecdotally, undated submissions can sit for up to a week before being noticed, though I suppose in the grand scheme of things that's not a huge delay.

The one thing I _would_ like to see is AFCH not using the timestamps in the /draft version to populate the /pending, as it has occasionally resulted in a "year old submission" right off the bat, but I think that's a separate issue.

**Comment by robertsky** [comment]
Ideally the timestamp of when the template was last inserted. But if it is undoable, the timestamp of when the cleaning up is done.

**Comment by primefac** [comment]
Yeah, I guess in thinking about it more the fact that is has been submitted means the timestamp should be from that edit. The /draft timestamp issue is another problem entirely.

**Comment by NovemLinguae** [comment]
Additional diff: https://en.wikipedia.org/w/index.php?title=Draft:Demolition_Ranch&diff=prev&oldid=1234778471
